UI/xdg-data/com.obsproject.Studio.metainfo.xml.in

```diff
@@ -30,16 +32,32 @@
   <url type="bugtracker">https://github.com/obsproject/obs-studio/issues</url>
   <url type="donation">https://obsproject.com/contribute</url>
   <url type="translate">https://crowdin.com/project/obs-studio</url>
+
+  <recommends>
+    <internet>always</internet>
+
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </recommends>
+
+  <supports>
+    <internet>offline-only</internet>
+  </supports>
+
   <screenshots>
     <screenshot type="default">
       <image>https://obsproject.com/assets/images/OBSDemoApp2610.png</image>
+      <caption>The OBS Studio main window with grey rectangles, and SMPTE color bars</caption>
```

Comment:
What are your thoughts on postetially changing the image to the one that is currently used on <https://obsproject.com>? (That would be <https://obsproject.com/assets/images/features-new/hero.png>)

Reply:
It's now using https://obsproject.com/assets/images/OBSDemoApp301Flathub.png